### ISSUE TITLE
Problem with theme?

### ISSUE THREAD

**Issue by thearabbit** [VladShcherbin/slickjs#1]
I trey install, and then copay `slick them` to project but it show

``` js
client/css/slick-theme/slick-theme.css: @font-face missing '{'
```

Good place for theme???
`client/compatibility` or `client/css`???

**Comment by thearabbit** [comment]
now I fix, but slick style not good like demo on website.
Have any styles???

**Comment by vladshcherbin** [comment]
@thearabbit okay, there was kinda an error with a default theme from the original package. I've updated the themes folder and the default one is working fine now.

I also added a [demo](http://slickjs-default-theme.meteor.com) app with a default theme, updated the readme and uploaded the example app [here](https://github.com/VladShcherbin/slickjs/tree/master/examples/slick-theme).

I hope, this will solve your problems.
